oci_mlflow/oci_object_storage.py

```diff
@@ -55,6 +58,67 @@ def parse_os_uri(uri: str):
     return bucket, ns, path
 
 
+def get_token_path():
+    """
+    Gets delegation token path.
+
+    Return
+    ------
+    str
+        The delegation token path.
+    """
+    token_path = (
+        DEFAULT_DELEGATION_TOKEN_PATH
+        if os.path.exists(DEFAULT_DELEGATION_TOKEN_PATH)
+        else os.environ.get(DELEGATION_TOKEN_PATH)
```

Comment:
This line can be changed to 
`os.environ.get(DELEGATION_TOKEN_PATH, DEFAULT_DELEGATION_TOKEN_PATH)`

Reply:
Hi Mayoor, we expect to return `token_path=None` if we are not running in the Data Flow (`os.path.exists(DEFAULT_DELEGATION_TOKEN_PATH` will return false in this case). When token_path is None, we will provide the current `default_signer()`.

Once the ads.common.auth.default_signer() has been modified with this similar change, we can remove the change here and simply call `default_signer()`.

Reply:
Yes, that makes sense.